apps/package.json

```diff
@@ -50,8 +50,8 @@
     "@code-dot-org/js-numbers": "0.1.0-cdo.0",
     "@code-dot-org/p5.play": "1.1.9-cdo",
     "@code-dot-org/piskel": "0.13.0-cdo.0",
-    "@kadira/react-storybook-addon-info": "^3.3.0",
-    "@kadira/storybook": "^2.33.0",
+    "@storybook/addon-info": "3.2.11",
+    "@storybook/react": "3.2.11",
```

Comment:
:grin: :tada: :+1: Thanks for taking care of this upgrade!  Hopefully this puts us a small step away from the latest release, 3.3.3.  I won't recommend making that additional step in this PR, because it looks like [v3.3.0](https://github.com/storybooks/storybook/releases/tag/v3.3.0) is a _beast_, despite the minor version bump.  It's going to be necessary for the storyshots addon though.  FYI @mehalshah.

Reply:
I was hoping to keep this separated into a different PR, but AFAIK there's no version of `@kadira/storybook` that supports webpack 2, and no version of `@storybook/react` that supports webpack 1.  So unfortunately it's tied to the webpack upgrade PR.